XCUITests/IntegrationTests.swift

```diff
@@ -22,10 +40,29 @@ class IntegrationTests: BaseTestCase {
         navigator.nowAt(BrowserTab)
     }
 
+    func testFxASyncHistory () {
+        // History is generated using the DB so go directly to Sign in
+        // Sign into Firefox Accounts
+        navigator.goto(BrowserTabMenu)
+        navigator.goto(FxASigninScreen)
+        waitforExistence(app.webViews.staticTexts["Sign in"], timeout: 10)
+        userState.fxaUsername = ProcessInfo.processInfo.environment["FXA_EMAIL"]!
+        userState.fxaPassword = ProcessInfo.processInfo.environment["FXA_PASSWORD"]!
+        navigator.performAction(Action.FxATypeEmail)
+        navigator.performAction(Action.FxATypePassword)
+        navigator.performAction(Action.FxATapOnSignInButton)
+        allowNotifications()
+
+        // Wait for initial sync to complete
+        navigator.nowAt(BrowserTab)
+        navigator.goto(SettingsScreen)
+        waitforExistence(app.tables.staticTexts["Sync Now"], timeout: 12)
```

Comment:
Here's another opportunity to reduce duplicate, with a `waitForInitialSync` method or similar. Also, I think we should try to be consistent with the timeout.